nexus/src/db/model.rs

```diff
@@ -631,16 +631,48 @@ pub struct Sled {
     pub ip: ipnetwork::IpNetwork,
     // TODO: Make use of SqlU16
     pub port: i32,
+
+    /// The last IP address provided to an Oxide service on this sled
+    pub last_used_address: IpNetwork,
 }
 
+// TODO-correctness: We need a small offset here, while services and
```

Comment:
[non-blocking] Would it make sense to bump the last used address when RSS is initializing these services?

Reply:
If you mean the offset, I'm not sure. I think it'll depend on the specifics of how RSS sets up the addresses as opposed to Nexus. For example, I think the current plan is for the RSS to _only_ allocate the prefix and the first 1-3 addresses. Those are for the sled server itself, plus possibly Nexus and CRDB.

If you mean the bump allocation strategy, I think that applies in either case.